### PR TITLE
Update ES versions in cloud services

### PR DESCRIPTION
**Pull request by srsathish92**
## Purpose of this pull request

This PR to update supported ES versions in cloud services based on ece-tools as a reference. 

## Affected DevDocs pages
https://devdocs.magento.com/cloud/project/project-conf-files_services.html

## Links to ECE Tools code

https://github.com/magento/ece-tools/blob/develop/src/Service/Validator.php#L52

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/project/project-conf-files_services.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ The following table lists the services used in {{site.data.var.ece}} and their v
 {:.error-table}
 Service   |  Magento 2.4  |Magento 2.3  | Magento 2.2
 --------- | ------------- |-------------| ------------
-`elasticsearch` | 7.5.x, 7.6.x, 7.7.x | **Magento version 2.3.5 and later**— 5.2, 6.5, 7.2, 7.5<br>**Magento version 2.3.1 to 2.3.4**— 6.5<br>**Magento version 2.3.0**— 2.4, 5.2  | **Magento version 2.2.8 and later**— 5.2, 6.5 <br>**Magento version 2.2.0 to 2.2.7**— 5.2
+`elasticsearch` | 7.4.x, 7.5.x, 7.6.x, 7.7.x | **Magento version 2.3.5 and later**— 5.2, 6.5, 6.8, 7.4, 7.5, 7.6, 7.7<br>**Magento version 2.3.1 to 2.3.4**— 5.2, 6.5<br>**Magento version 2.3.0**— 5.2  | **Magento version 2.2.8 and later**— 5.2, 6.5 <br>**Magento version 2.2.0 to 2.2.7**— 5.2
```
Comment (NadiyaS) [inline src/cloud/project/project-conf-files_services.md:164]: Please remove ES 7.4 version from the list. Magento Cloud does not support this version for now

Comment (meker12) [inline src/cloud/project/project-conf-files_services.md:164]:
```suggestion
`elasticsearch` | 7.5.x, 7.6.x, 7.7.x | **Magento version 2.3.5 and later**— 5.2, 6.5, 6.8, 7.5, 7.6, 7.7<br>**Magento version 2.3.1 to 2.3.4**— 5.2, 6.5<br>**Magento version 2.3.0**— 5.2  | **Magento version 2.2.8 and later**— 5.2, 6.5 <br>**Magento version 2.2.0 to 2.2.7**— 5.2
```

Comment (srsathish92) [reply src/cloud/project/project-conf-files_services.md]: @meker12 Updated, can you please review now?

Comment (srsathish92) [reply src/cloud/project/project-conf-files_services.md]: @NadiyaS Thanks for pointing out. Remove ES 7.4, Can you please review now?

**Comment by meker12** [comment]
running tests

**Pull request merged**